libs/components/layout/src/lib/modules/box/box-header.component.html

```diff
@@ -1,3 +1,3 @@
-<div class="sky-box-header">
+<div class="sky-box-header sky-control-help">
```

Comment:
The header component should use an element with the `sky-control-help-container` and an `ng-content select=".sky-control-help"` like our other components. See [description list](https://github.com/blackbaud/skyux/blob/main/libs/components/layout/src/lib/modules/description-list/description-list-term.component.html#L3) for an example.

Reply:
Updated this to use `sky-control-help-container`.